rest_framework/authtoken/models.py

```diff
@@ -1,26 +1,22 @@
 import binascii
 import os
 
-from django.conf import settings
```

Comment:
We still need settings to test whether or not the authtoken is in the INSTALLED_APPS.